test/connect-backbone-to-react.test.js

```diff
@@ -96,7 +96,7 @@ describe('connectBackboneToReact', function() {
     });
 
     afterEach(function() {
-      wrapper.unmount();
+      if (wrapper.exists()) wrapper.unmount();
```

Comment:
Why is this now required? What changed? Seems like a bandaid when there is some other underlying issue at hand?

Reply:
So there are 2 tests which unmounts inside the test so when the afterEach tries to run the unmount again it fails. That's why I will just keep this if check in the afterEach and I have removed from the individual tests. 

When removed 2 tests results in an error ` Error: Method “unmount” is meant to be run on 1 node. 0 found instead.` we have upgraded to 3.10 and I found this in the enzyme issues https://github.com/airbnb/enzyme/issues/1880

Reply:
Aha, got it.